### ISSUE TITLE
Standard Build VMs for Linux

### ISSUE THREAD

**Issue by stuarteberg** [ilastik/ilastik#925]
I usually build the Linux binaries using the [ilastik-test-vm][1], which is based on Ubuntu 12.04.  Unfortunately, it seems that many scientists use variants of RHEL 6.4 (e.g. CentOS or ScientificLinux).  The standard Linux binary we're shipping doesn't support those platforms.  (The main issue seems to be the glibc version -- 2.12 vs 2.15)

We should cook up a standard VM, preferably via vagrant, that can generate Linux binaries compatible with both RHEL and Debian.

[1]: https://github.com/ilastik/ilastik-test-vm

**Comment by stuarteberg** [comment]
I created a CentOS 5.11 VM, configured as described in the [ilastik-build-conda readme](https://github.com/ilastik/ilastik-build-conda#linuxvm).  I exported it from VirtualBox in the standard `.ova` format.  With this VM, we can build ilastik binaries for just about any Linux release we're likely to run into.

Anybody have any ideas about where we should store this thing?  It's pretty big (15GB uncompressed), so it probably shouldn't go on files.ilastik.org.  Somewhere on `hci07`, perhaps?
